### PR TITLE
Downgrade JS

### PR DESCRIPTION
**Pull request by nLight**
Usage of `let` causes trouble with `webpack@1` `uglifyjs` plugin.

### CODE REVIEW

**Review by Poltergeist — CHANGES_REQUESTED** [review]
As `var` has a different scope then `let` and `const` I would advise to change the way the variables are defined, I would opt to define all the variables at the top of their scope which would be the function. Just to omit the possibility that one variable could be defined multiple times and because the `var` declaration is hoisted anyway so we are not changing anything code execution wise if we place all the `var`s at the beginning of the scope. It makes this only more visible.

**Comment by nLight** [comment]
@Poltergeist totally 👍  please have another look

**Review by juliangieseke — APPROVED** [review]

**Review by orlandohohmeier — APPROVED** [review]

File: src/read.js [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
 module.exports = function read(input) {
-  const records = [];
-  let rest = input;
+  var records = [];
+  var rest = input;
+  var delimiterPosition,
+    recordLength,
+    recordStartPosition,
+    recordEndPosition,
+    record;
```
Comment (Poltergeist) [inline src/read.js]: 👍 This is what meant.

**Review by Poltergeist — APPROVED** [review]
👍 Let's get this merged.

**Pull request merged**